resources/views/outlets/show.blade.php

```diff
@@ -21,7 +21,11 @@
                 @can('update', $outlet)
                     <a href="{{ route('outlets.edit', $outlet) }}" id="edit-outlet-{{ $outlet->id }}" class="btn btn-warning">{{ __('outlet.edit') }}</a>
                 @endcan
-                <a href="{{ route('outlets.index') }}" class="btn btn-link">{{ __('outlet.back_to_index') }}</a>
+                @if(\Illuminate\Support\Facades\Auth::check())
```

Comment:
@aznadesign, I believe it is better we use `auth()` instead of a full namespaced class facade.
```suggestion
                @if(auth()->check())
```